app/src/main/res/values-es/strings.xml

```diff
@@ -1,272 +1,420 @@
-    <string name="submit_loan">Submit Loan</string>
-    <string name="new_loan_application">New Loan Application for </string>
-    <string name="update_loan_application">Update Loan Application for </string>
-    <string name="loan_interest_type">Interest Type</string>
-    <string name="amortization">Amortization</string>
-    <string name="interest_calculation_period">Interest calculation period</string>
-    <string name="repayment_strategy">Repayment Strategy</string>
-    <string name="pay_to">Pay To</string>
-    <string name="pay_from">Pay From</string>
-    <string name="cancel">Cancel</string>
-    <string name="review_transfer">Review</string>
-    <string name="transfer_to">Transfer To</string>
-    <string name="transfer_from">Transfer From</string>
-    <string name="making_transfer">Making Transfer</string>
-    <string name="deposit">Deposit</string>
-    <string name="enter_amount">Enter Amount</string>
-    <string name="remark_is_mandatory">Remark is mandatory</string>
-    <string name="approved">Approved</string>
-    <string name="shares_pending">Pending</string>
-    <string name="loan_amount_paid">Paid</string>
-    <string name="balance">Balance</string>
-    <string name="rejected">Rejected</string>
-    <string name="waiting">Waiting</string>
-    <string name="overpaid">Overpaid</string>
+    <string name="login">Iniciar sesión</string>
+    <string name="toast_welcome">Hola, %1$s.</string>
+    <string name="unable_to_connect">Falta conexión internet</string>
+    <string name="basic">Primario</string>
+    <string name="username">Nombre de usuario</string>
+    <string name="progress_message_login">Acceder</string>
+    <string name="progress_message_loading">Carga</string>
+    <string name="password">Contraseña</string>
+    <string name="email">Dirección de correo electrónico</string>
+    <string name="accounts">Cuentas</string>
+    <string name="clients">Clientes</string>
+    <string name="funds_transfer">Transferencia de fondos</string>
+    <string name="recent_transactions">Transacciones recientes</string>
+    <string name="charges">Honorarios</string>
+    <string name="questionnaire">Cuestionario</string>
+    <string name="about_us">Sobre nosotros</string>
+    <string name="saving_account_details">Guardar detalles de la cuenta</string>
+    <string name="savings_account">Cuenta de ahorros</string>
+    <string name="error_loan_account_details_loading">Error al cargar la información de la cuenta de crédito</string>
+    <string name="nominal_interest_rate">Tasas de interés nominales</string>
+    <string name="account_number">Numero de cuenta</string>
+    <string name="account_balance">Saldo de cuenta</string>
+    <string name="total_deposits">Depósitos totales</string>
+    <string name="open_drawer">Abre el panel de navegación</string>
+    <string name="close_drawer">Cerrar el panel de recuperación</string>
```

Comment:
Why did you use different words for the same translation? "navegación" and "recuperación"?

Reply:
Looks like a silly mistake, thanks for pointing it out!